### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by ahzia**
In this Milestone, I have set up a simple new React App.
Kindly check it and Provide Your Feedback.
Note:
I removed the linter's configuration as it was preventing the app from starting, when Running `npm start` is was showing me an Error "Failed to load config "Airbnb" to extend from".

### CODE REVIEW

**Review by arslanbisharat — APPROVED** [review]
# Invalid submission 

Hi,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has filing linters.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by AshakaE — COMMENTED** [review]
### Hi @ahzia  🙋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request ❌ 

You have submitted a project, that has failing linter checks 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by ahzia** [comment]
@arslanbisharat and @AshakaE Thanks for the Review,
I solved the linters issue but I had an issue with my React Project, when I added the linters configuration I was not able to run my project using `npm start` It was showing me an error, "Failed to load config "Airbnb" to extend from".
this made me delete my linters configuration.
Can You give me any hint on how I can solve it?
Thanks

File: README.md [hunk 1]
```diff
@@ -1,2 +1,70 @@
...
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.\
+You will also see any lint errors in the console.
+
+### `npm test`
```
Comment (henatan99) [inline README.md]: - [x] As per the general requirement you need to update your readme using the [Microverse readme template](https://github.com/microverseinc/readme-template) by customizing it to your project. Even though it is just a setup, you can have a minimal description like a one-line about sentence, instructions, your links, acknowledgment, ... Kindly update it. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @ahzia , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ahzia) [reply README.md]: @henatan99 Thanks for the Review,
I had an issue with my linters which I mentioned in my comments for other code Reviewers, Can you check if you know how I can solve it.

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @ahzia 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewers.

 #### Optional suggestions
    
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**